File: WorldFeed/Server/src/WorldFeed.Common/DTO/Webhooks/WebhookEnvironmentDTO.cs
namespace WorldFeed.Common.DTO.Webhooks
{
    using Newtonsoft.Json;

    using WorldFeed.Common.Public.Models.Interfaces.DTO.Webhooks;

    public class WebhookEnvironmentDTO : IWebhookEnvironmentDTO
    {
        [JsonProperty("environment_name")]
        public string Name { get; set; }

        public IWebhookDTO[] Webhooks { get; set; }
    }
}

File: Sharebook/Client/src/app/core/Core/DTO/Webhooks/WebhookEnvironmentDTO.ts
import {IWebhookDTO} from "../../../Public/Models/Interfaces/DTO/Webhooks/IWebhookDTO";
import {IWebhookEnvironmentDTO} from "../../../Public/Models/Interfaces/DTO/Webhooks/IWebhookEnvironmentDTO";

export class WebhookEnvironmentDTO implements IWebhookEnvironmentDTO {
  // [JsonProperty("environment_name")]
  public name: string;

  public webhooks: IWebhookDTO[];
}
